itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java

```diff
@@ -458,10 +458,10 @@ private void verifyReplTargetProperty(Map<String, String> props) {
     assertTrue(props.containsKey(ReplConst.REPL_TARGET_TABLE_PROPERTY));
   }
 
-  public void verifyTargetOfReplProperty(String dbName) throws Exception {
+  public boolean isSetTargetOfReplProperty(String dbName) throws Exception {
     Database db = getDatabase(dbName);
-    assertTrue(db.getParameters().containsKey(ReplUtils.TARGET_OF_REPLICATION));
-    assertTrue(Boolean.getBoolean(db.getParameters().get(ReplUtils.TARGET_OF_REPLICATION)));
+    return db.getParameters().containsKey(ReplUtils.TARGET_OF_REPLICATION)
+            && !db.getParameters().get(ReplUtils.TARGET_OF_REPLICATION).isEmpty();
```

Comment:
Isn't the value always boolean if set? Why are we breaking that assumption?

Reply:
This property is set if its value is non empty.